app/web/src/components/WorkspaceExportModal.vue

```diff
@@ -60,8 +58,8 @@ function open() {
   openModal();
 }
 
-async function continueHandler() {
-  await moduleStore.EXPORT_WORKSPACE();
+function continueHandler() {
+  moduleStore.EXPORT_WORKSPACE();
```

Comment:
will re-enable the no floating promises eslint rule soon, but unrelated to this, so leaving it as is to match other similar uses of api actions.